Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM funkyfuture/nest-of-serpents
+
+ENTRYPOINT tox
+WORKDIR /src
+
+RUN apt-get -q update \
+ && DEBIAN_FRONTEND=noninteractive apt-get install -qy \
+        make \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/* \
+ && pyp install flake8 pytest tox PyYAML Sphinx \
+ && mkdir /home/tox \
+ && mv /root/.cache /home/tox/
+
+# this will be set to the user's id who's running a script that uses this
+# image in order to have file-ownerships on the host-system
+# at some point there should be native Docker-implementation for this
+ENV TOX_USER_ID=1000
```

Comment:
no, it gets substitued in the `run-docker-tests` on each invokation to match file-ownerships with a possible `.tox`-folder.